lib/fluent/plugin/exception_detector.rb

```diff
@@ -59,7 +59,7 @@ def self.supported
     ].freeze
 
     PYTHON_RULES = [
-      rule(:start_state, /Traceback \(most recent call last\)/, :python),
+      rule(:start_state, /^Traceback \(most recent call last\):$/, :python),
```

Comment:
@qingling128 Yes, let's open a FR internally to allow customizing the regexps...